### ISSUE TITLE
Empty Interfaces List provided by API bricks Inventory Script

### ISSUE THREAD

**Issue by Mawiguk0** [ansible-collections/community.zabbix#826]
##### SUMMARY
The inventory script under /scripts/inventory/zabbix.py stops working because of a missing check for a empty list provided by the zabbix api. 

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
https://github.com/ansible-collections/community.zabbix/blob/0713b4179d3f31c9a68769a65728bc18ee5e6896/scripts/inventory/zabbix.py#L106
https://github.com/ansible-collections/community.zabbix/blob/0713b4179d3f31c9a68769a65728bc18ee5e6896/scripts/inventory/zabbix.py#L142

##### ANSIBLE VERSION

```
ansible 2.10.8
  config file = /home/user/git/ansible/dev/ansible.cfg
  configured module search path = ['/home/user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.10.6 (main, Aug 10 2022, 11:40:04) [GCC 11.3.0]
```

##### CONFIGURATION

```
DEFAULT_HOST_LIST(/home/user/git/ansible/devices/ansible.cfg) = ['/home/user/git/ansible/devices/zabbix.py']
HOST_KEY_CHECKING(/home/user/git/ansible/devices/ansible.cfg) = False
```

##### STEPS TO REPRODUCE

Just use the zabbix api with e.g. 

```bash
ansible-inventory --inventory zabbix.py --list
```

##### EXPECTED RESULTS

A complete Inventory file.

##### ACTUAL RESULTS
Script stops working and breaks because of a index out of range error.

**Comment by Mawiguk0** [comment]
Mawiguk0:main provided a bugfix for this

**Comment by D3DeFi** [comment]
Fixed by #827
